packages/scene-composer/src/layouts/SceneLayout/components/ScenePanel.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import ScenePanel from './ScenePanel';
+import { Direction } from './utils';
+
+jest.mock('./FoldableContainer', () => (props) => <div data-testid='FoldableContainer' {...props} />);
+jest.mock('./TabbedPanelContainer', () => (props) => <div data-testid='TabbedPanelContainer' {...props} />);
+
+describe('<ScenePanel />', () => {
+  it('should render the left panel as closed by default', () => {
+    const leftArgs = {
+      direction: Direction.Left,
+      panels: { [1]: <div>TestLeftPanel</div> },
+    };
+    const container = render(<ScenePanel {...leftArgs} />);
+    expect(container).toMatchSnapshot();
+  });
+  it('should render the right panel as closed by default', () => {
+    const rightArgs = {
+      direction: Direction.Right,
+      panels: { [1]: <div>TestRightPanel</div> },
+    };
+    const container = render(<ScenePanel {...rightArgs} />);
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
Instead of relying on snapshots, is it possible to assert on the presence of the `open` class name?

Reply:
Fixed!